src/core_plugins/input_control_vis/public/components/vis/range_control.js

```diff
@@ -29,17 +29,26 @@ import {
   EuiFlexItem,
 } from '@elastic/eui';
 
-const toState = (props) => {
+const toState = ({ control }) => {
+  let sliderValue = control.value;
+  if (!control.hasValue()) {
+    // InputRange component does not have an "empty state"
+    // Faking an empty state by setting the slider value range to length of zero anchored at the range minimum
+    sliderValue = {
+      min: control.min,
+      max: control.min
+    };
+  }
   const state = {
-    sliderValue: props.control.value,
+    sliderValue: sliderValue,
```

Comment:
nit: can be shortened to just `sliderValue` (https://github.com/airbnb/javascript#es6-object-concise)